scripts/build.py

```diff
@@ -0,0 +1,39 @@
+from utils import (
+    get_manim_plugins,
+    parse_examples_folder,
+    write_plugins_to_json,
+    write_governance_file,
+    render_manim_example,
+    write_example_json,
+    guarantee_existence_of_folders,
+)
+from cli import parser
+
+config = parser.parse_args()
+guarantee_existence_of_folders()
+
+MANIM_PLUGINS = []
+MANIM_PLUGINS = get_manim_plugins(default_plugins="default.json")
+write_plugins_to_json(manim_plugins=MANIM_PLUGINS)
+write_governance_file()
+
+examples = parse_examples_folder()
+# If None of the examples are marked as visible
+# make sure to add the first one as visible so
+# that atleast one example is visible.
+for example in examples:
```

Comment:
Since you're trying to refactor this as modules, why not wrap all these codes under a check for whether the code is directly invoked? (`if __name__ == '__main__':`)

Reply:
I added a `guarentee_one_visible` function as well.
And the functions that actually do the building are called under the `if __name__ == '__main__':` check.